shared-lib/spiutils/src/protocol/firmware.rs

```diff
@@ -262,9 +259,6 @@ impl ToWire for UpdatePrepareRequest {
 wire_enum! {
     /// The result of an update prepare request.
     pub enum UpdatePrepareResult: u8 {
-        /// Unknown result type.
-        Unknown = 0xff,
-
         /// Success
         Success = 0x00,
```

Comment:
Would it make more sense for Success=1 and Failure=2, to more easily catch uninitialized values?  Or is that impossible in rust?

Reply:
Rust wouldn't allow uninitialized values. You could argue that byte "0" on the wire is worse than byte "1", but for that we have the CRC to catch garbage. I'd leave it as is.